bokeh/command/subcommands/serve.py

```diff
@@ -682,6 +698,16 @@ class Serve(Subcommand):
             type    = int,
         )),
 
+        ('--session-expiration', dict(
+            metavar = 'N',
+            action  = 'store',
+            help    = "Number of seconds before which a newly created "
+                      "session expires if no Websocket connection has "
+                      "been made (defaults to 300).",
```

Comment:
We might need more precise terminology here. The possible confusion for the name is thinking that this controls how long until a successfully session expires after it starts. Maybe `session-token-expiration`? And maybe the description can be more direct (e.g. no "before which" :D) How about "Duration in seconds that a session token is valued. After the expiry time has elapsed, the token will not be able create a new session"